frontend/src/App.js

```diff
@@ -52,6 +57,7 @@ function App() {
             <Search
               searchResults={searchResults}
               searchStatus={searchStatus}
+              searchQuery={searchQuery}
```

Comment:
My IDE is saying the the navigate variable should be add to the useEffect dependency array

Reply:
done